Cargo.toml

```diff
@@ -74,3 +74,4 @@ thiserror = "1.0"
 thiserror-no-std = "2.0"
 tokio = "1.39"
 trybuild = "1"
+cargo_metadata = "0.18"

```

Comment:
Please see the comment saying it has to be in alphabetical order. We also prefer crates to be named in kebab-case instead of snake one